src/dialect/mysql.rs

```diff
@@ -51,7 +51,7 @@ impl Dialect for MySqlDialect {
     }
 
     fn is_identifier_part(&self, ch: char) -> bool {
-        self.is_identifier_start(ch) || ch.is_ascii_digit()
+        self.is_identifier_start(ch) || ch.is_ascii_digit() || !ch.is_ascii()
```

Comment:
Can we add a comment to these highlighting that they implement unicode character support?